aggregator/okex.go

```diff
@@ -0,0 +1,54 @@
+package aggregator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+const (
+	okexPriceUrl = "https://www.okex.com/api/v5/market/ticker?instId=%s%s%s"
+)
+
+type okexPriceRequest struct {
+	Data []okexTicker
+}
+
+type okexTicker struct {
+	Price string `json:"last"`
+}
+
+type okex struct {
+	ResponseGetter
+}
+
+// FetchPrice will fetch the price using the http client
+func (o *okex) FetchPrice(ctx context.Context, base string, quote string) (float64, error) {
+	if strings.Contains(strings.ToUpper(quote), QuoteUSDFiat) {
```

Comment:
we should extract this in a base component

Reply:
can be done in an upcoming PR

Reply:
added TODO, will be added in the next PR once all the components are merged